news/views.py

```diff
@@ -526,6 +526,15 @@ def newsletters(request):
     })
 
 
+@require_safe
+@cache_page(30)
+def healthz(request):
+    """for use with healthchecks. wrapped with `cache_page` to test redis cache."""
+    # here to make sure DB is working
+    Newsletter.objects.exists()
```

Comment:
perhaps it would be useful to make this an assertion so that it throws a 500 if it gets an empty result?